### PR TITLE
Link OSS

### PR DESCRIPTION
**Pull request by ccapurso**

### CODE REVIEW

File: vault/hcp_link/capabilities/meta/meta.proto [hunk 1]
```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
```
Comment (hghaf099) [inline vault/hcp_link/capabilities/meta/meta.proto:1]: Just raising an idea here. We have a subdirectory for node status protobuf file. Would it make sense to move all protobuf files for other capabilities there too?

File: vault/hcp_link/link.go [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+		logger:       logger,
+	}
+
+	// Start hcpLink and ScadaProvider
+	err = hcpLinkVault.start()
+	if err != nil {
+		return nil, fmt.Errorf("failed to start hcp link, %w", err)
+	}
+
+	wrapped := &WrappedHCPLinkVault{hcpLinkVault}
```
Comment (hghaf099) [inline vault/hcp_link/link.go:140]: I wonder if we still need this wrapped struct. IIRC, we used it to be able to introduce a stub for OSS. Now, that these are ported over, do we still need this?

**Comment by hghaf099** [comment]
Looks great! Just posted a couple refactoring comments.

File: vault/hcp_link/link.go (hunk 2 above)
Comment (ccapurso) [reply vault/hcp_link/link.go:140]: That's a really good point. I don't think so.

**Comment by ccapurso** [comment]
> Seems like the no-changelog label is added. I guess this is because we don't have an official one for it yet?
Yup, I think we have one now. I'll add one shortly.

File: changelog/18228.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+hybrid-management: Add OSS support for hybrid management
```
Comment (hghaf099) [inline changelog/18228.txt:2]: `hybrid-management` is neat! 
Are you going to change `hcplink` references in the code as well?

**Review by hghaf099 — APPROVED** [review]
Looks good! I just have one last question in the change log.

File: changelog/18228.txt (hunk 3 above)
Comment (ccapurso) [reply changelog/18228.txt:2]: Possibly, but maybe in a separate PR. We might want to wait until we align on something.

**Pull request merged**